src/main/java/org/maxgamer/rs/core/Core.java

```diff
@@ -227,15 +163,7 @@ public static ConsoleSender getConsole() {
      *
      * @return the server
      */
-    public static synchronized Server getServer() {
-        if(server == null) {
-            try {
-                server = new Server(); //Binds port port
-            } catch (Exception e) {
-                LOGGER.error("There was an error instantiating the server", e);
-                System.exit(3);
-            }
-        }
+    public static Server getServer() {
         return server;
     }
 
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS1166)